### ISSUE TITLE
SVT:  api doc is not properly formatted in MS Edge

### ISSUE THREAD

**Issue by tam512** [OpenLiberty/open-liberty#2445]
-  Viewing api doc /openapi in MS Edge shows unformatted document.  This is the same as in Internet Explorer.  In openapi-3.0 release, we documented this restriction for IE  at
https://www-03preprod.ibm.com/support/knowledgecenter/SSAW57_liberty_test/com.ibm.websphere.wlp.nd.multiplatform.doc/ae/rwlp_restrict.html#rwlp_restrict__openapi30
- If this is the same limitation with Edge, we need to document it for mpOpenAPI-1.0
![image](https://user-images.githubusercontent.com/22480038/36742807-3c740c1c-1bae-11e8-98b6-530cd3d0e96f.png)

**Comment by mrglavas** [comment]
@tam512 Agreed, it looks like Edge has the same limitation as IE and we should document it for mpOpenAPI-1.0.

**Comment by mrglavas** [comment]
Document as a limitation and close.

**Comment by arturdzm** [comment]
Found actual fix for this

**Comment by mrglavas** [comment]
@arturdzm Please close the issue once the fix is in the stream.

**Comment by tam512** [comment]
I also verified that /api/docs and /openapi are formatted ok on MS Edge using build 20180530-0500.33
